### DISCUSSION TITLE
Duration of "chia plots check" is huge

### DISCUSSION THREAD

**Discussion by happycouak** [Chia-Network/chia-blockchain#4596] [Q&A] [answered]
Checking my plots (77) took ~36 minutes, which indicate the proof need ~46sec to be verified.

I can see an average of ~ 120 iops on the hard drive (non ssd), which is the maximum the drive can sustain.

I am a bit worried if a plot is elected but the time to check proofs is so long that it missed the chance.

Is this an issue or the real workload when a plot is elected is different ?

Also how can I check the time used when a plot proof is checked ?

**Comment by lvcivs** [comment, marked as answer]
The command `chia plots check` will check each plot repeatedly, I believe around 30 times for each plot by default. While farming you will see a much lighter load. It's normal that the `chia plots check` takes a long time and this is nothing to worry about.

Regarding your last question, you need to change the log level to INFO in your `config.yaml` and restart the node. After that you will find the details about each attempted proof in the `debug.log` file.

**Reply by happycouak** [reply]
Many thanks.

**Reply by lvcivs** [reply]
You are welcome. If you feel your question has been fully answered, you can select this answer as the 'correct' one and this issue will appear as solved.

**Reply by maxbelz** [reply]
another question how to output chia check command to file? i have 1500 plots... it is crazy to read an output in console
